Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -3634,6 +3634,25 @@ - (void)runActionNamed:(NSString *)name
                 }];
             });
             return;
+        } else {
+            // Message doesn't seem to be on the device,
+            // so let's forceContentUpdate and retry showing it.
+            [Leanplum forceContentUpdate: ^(void) {                
+                LPActionContext *chainedActionContext =
```

Comment:
This is an exact copy of above. Please use blocks. Block is essentially a function that you can declare within.
It's critical to obj-c development. It's also what you are passing in forceContentUpdate.

To see how it is used in practice look here:
https://github.com/Leanplum/Leanplum-iOS-SDK/blob/master/Leanplum-SDK/Classes/LPMessageTemplates.m#L375
https://github.com/Leanplum/Leanplum-iOS-SDK/blob/master/Leanplum-SDK/Classes/LPNetworkOperation.m#L160

Syntax is rather unique, check this site for reference:
http://fuckingblocksyntax.com/

Reply:
Fixed.